### PR TITLE
Added a few test cases for the week view

### PR DESCRIPTION
**Pull request by rama1990br**

### CODE REVIEW

File: calendar.css [hunk 1]
```diff
@@ -20,18 +20,22 @@
...
-    /* background: red; */
-    float: left;}
-.column-header-text{}
-.column-header-text{}
-.week-rows-container{}
-.each-row-container{}
-.each-row{}
-.each-cell{}
-.each-cell-header{}
+.div-table {
```
Comment (hrishikeshs) [inline calendar.css]: please do not use names like `div-table`, css class names should be semantic. An example would be, for a calendar application, you can call this:
```
. calendar {
  ... properties...
}
```
`div-table` conveys no information about what this class styles, on the other hand, calling it `calendar` conveys useful information.

File: calendar.css [hunk 2]
```diff
@@ -20,18 +20,22 @@
...
+  display: table;
+  width: auto;
+  background-color: #eee;
+  border: 1px solid #666666;
+  border-spacing: 5px; /* cellspacing:poor IE support for  this */
+}
+.div-table-row {
```
Comment (hrishikeshs) [inline calendar.css]: Please follow the style guide we are using: http://codeguide.co/

File: calendar.css [hunk 3]
```diff
@@ -20,18 +20,22 @@
...
+  display: table-row;
+  width: auto;
+  clear: both;
+}
+.div-table-cell {
+  float: left; /* fix for  buggy browsers */
```
Comment (hrishikeshs) [inline calendar.css]: what does "float" do?

File: calendar.css [hunk 4]
```diff
@@ -20,18 +20,22 @@
...
-.each-cell-header{}
+.div-table {
+  display: table;
+  width: auto;
+  background-color: #eee;
+  border: 1px solid #666666;
+  border-spacing: 5px; /* cellspacing:poor IE support for  this */
+}
+.div-table-row {
+  display: table-row;
```
Comment (hrishikeshs) [inline calendar.css]: why this rule?

File: month-view.html [hunk 5]
```diff
@@ -13,15 +13,12 @@
...
-    <div id="month-view">
-      <div role="main" class="main-content">
-        <div role="grid" class="grid-content">
-          <div role="row" id="weekOfDayHeader" class="week-of-day-header">
-          </div>
-          <div id="rowHolder" role="presentation" class="week-rows-container">
-          </div>
+    <div id="monthName"></div>
+    <div id="tableContainer" class="div-table">
+      <!--<div role="row" id="weekOfDayHeader" class="week-of-day-header">
```
Comment (hrishikeshs) [inline month-view.html]: delete the commented out code.

File: calendar.css [hunk 6]
```diff
@@ -20,18 +20,22 @@
...
-    float: left;}
-.column-header-text{}
-.column-header-text{}
-.week-rows-container{}
-.each-row-container{}
-.each-row{}
-.each-cell{}
-.each-cell-header{}
+.div-table {
+  display: table;
```
Comment (hrishikeshs) [inline calendar.css]: what does display do? what are the other values for this property?

File: calendar.css (hunk 1 above)
Comment (rama1990br) [reply calendar.css]: Have changed the class names accordingly, please check and let me know whether they are okay.

File: calendar.css (hunk 2 above)
Comment (rama1990br) [reply calendar.css]: In the process of implementing it

File: month-view.html [hunk 7]
```diff
@@ -13,15 +13,12 @@
...
     <script src="./Date.js"></script>
     <script src="./month-view.js"></script>
-    <div id="month-view">
-      <div role="main" class="main-content">
-        <div role="grid" class="grid-content">
-          <div role="row" id="weekOfDayHeader" class="week-of-day-header">
-          </div>
-          <div id="rowHolder" role="presentation" class="week-rows-container">
-          </div>
+    <div id="monthName"></div>
```
Comment (hrishikeshs) [inline month-view.html]: nipick: hyphens instead of camelCase

**Review by hrishikeshs — COMMENTED** [review]
Looks good, please fix the style issues and push to master.

**Review by hrishikeshs — APPROVED** [review]
This is great work!! 🎉 🎉 🎉

**Pull request merged**